stable/kube-slack/templates/deployment.yaml

```diff
@@ -18,6 +18,12 @@ spec:
       labels:
         app: {{ template "kube-slack.name" . }}
         release: {{ .Release.Name }}
+      {{- if .Values.annotations }}
+      annotations:
+      {{- range $key, $value := .Values.annotations }}
+        {{ $key }}: {{ $value | quote }}
+      {{- end }}
+      {{- end }} 
```

Comment:
Can you instead use the `toYaml` function? See https://github.com/helm/charts/blob/bf484e940f3b5abdb8e0d2203e46e4e6628ddec6/stable/spinnaker/templates/ingress/deck.yaml#L6-L7 for an example

Reply:
Done

Reply:
I think that the indentation spaces before `{{ toYaml .Values.annotations | indent 4 }}` need to be removed, as this is done already by the `indent`. This currently doubles the indentation for the first line.
See [here](https://github.com/helm/charts/blob/master/stable/postgresql/templates/statefulset.yaml#L37) for an example.

Reply:
Done